typescript-test-samples/schema-and-contract-testing/tests/schema-testing.test.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+import jsonSchemaDiff from 'json-schema-diff';
+
+// The schema can be obtained from a repository or from Amazon EventBridge Schema Registry (the latter supports only OpenAPI 3.0 and Json Schema Draft 4 versions).
+// Here we're just reading schemas from a local json file for simplicity.
+import schema_v1_0_0 from '../schemas/Json/CustomerCreated-v1.0.0.json';
+import schema_v1_1_0 from '../schemas/Json/CustomerCreated-v1.1.0.json';
+import schema_v1_2_0 from '../schemas/Json/CustomerCreated-v1.2.0.json';
+import schema_v1_3_0 from '../schemas/Json/CustomerCreated-v1.3.0.json';
+
+test('adding new optional elements is backward compatible', async () => {    
```

Comment:
Same as above... we've been using `describe` blocks as the root container for tests to add context and help with grouping. Could we do so here as well?

Reply:
done